addons/binding/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/internal/model/OSHISysteminfo.java

```diff
@@ -478,7 +482,7 @@ public DecimalType getSwapUsedPercent() {
     }
 
     private long getSizeInMB(long sizeInBytes) {
-        return sizeInBytes /= 1024 * 1024;
+        return sizeInBytes / 1048576; // 1024 * 1024
```

Comment:
I would just keep the numbers. The fact that you have to put in a comment to explain says it all. The `/=` is indeed wrong.
I've also added a round because the current does a floor. So `1.9` becomes `1`, while it's probably nicer to see `2`. 
Note I've added `D` to the first const otherwise the expression will be calculated as long and implicitly floored.
```suggestion
        return Math.round(sizeInBytes / (1024D * 1024));
```